ruma-events/tests/room_message.rs

```diff
@@ -225,22 +225,20 @@ fn edit_deserialization_future() {
 
     assert_matches!(
         from_json_value::<MessageEventContent>(json_data).unwrap(),
-        MessageEventContent::Text(TextMessageEventContent {
-            body,
-            formatted: None,
+        MessageEventContent {
+            msgtype: MessageType::Text(TextMessageEventContent {
+                body,
+                formatted: None,
+            }),
             relates_to: Some(Relation::Replacement(Replacement { event_id })),
             new_content: Some(new_content),
-            ..
-        }) if body == "s/foo/bar"
+        } if body == "s/foo/bar"
             && event_id == ev_id
             && matches!(
                 &*new_content,
                 MessageEventContent::Text(TextMessageEventContent {
                     body,
                     formatted: None,
-                    relates_to: None,
-                    new_content: None,
-                    ..
```

Comment:
You removed a rest pattern here, does this actually compile? It really shouldn't.